src/Symfony/Component/Serializer/Normalizer/UidNormalizer.php

```diff
@@ -22,9 +22,9 @@ final class UidNormalizer implements NormalizerInterface, DenormalizerInterface,
     public const NORMALIZATION_FORMAT_KEY = 'uid_normalization_format';
 
     public const NORMALIZATION_FORMAT_CANONICAL = 'canonical';
-    public const NORMALIZATION_FORMAT_BASE_58 = 'base_58';
-    public const NORMALIZATION_FORMAT_BASE_32 = 'base_32';
-    public const NORMALIZATION_FORMAT_RFC_4122 = 'rfc_4122';
+    public const NORMALIZATION_FORMAT_BASE58 = 'base58';
+    public const NORMALIZATION_FORMAT_BASE32 = 'base32';
+    public const NORMALIZATION_FORMAT_RFC4122 = 'rfc4122';
```

Comment:
i feel like we should move the consts to `AbstractUid`

Reply:
Why? `AbstractUid` nor the Uid component wouldn't use them anywhere.

Reply:
because the formats are leaking now: `The ULID output format: base32, base58 or rfc4122', 'base32`

Reply:
I think these consts in Serializer are legit.
Then, the duplication in Uid is legit also - these are from different domains.
Also, the commands (on the cli), use inline values.
In the end, these consts in Uid won't serve any purpose for the devs.
They would actually make the public API of the component more complex to me.
I'd prefer not adding them as you might have understood:)

Reply:
i think both serializer+console are duplicating what could be `AbstractUid::toFormat($format)`

Reply:
oh, then I'd recommend splitting this discussion in a separate PR or issue.